squidb/src/com/yahoo/squidb/data/ValuesStorage.java

```diff
@@ -179,6 +179,18 @@ public void put(@Nonnull String key, @Nullable Object value, boolean errorOnFail
     }
 
     @Override
+    public String toString() {
+        StringBuilder result = new StringBuilder();
+        for (String key : keySet()) {
```

Comment:
Minor suggestion: you can use `valueSet` to iterate over the key/value pairs all at once without needing to call `get(key)` for each key.